src/open_inwoner/utils/time.py

```diff
@@ -40,5 +40,5 @@ def has_new_elements(
     return any(is_new(elem, attribute_name, delta) for elem in collection)
 
 
-def convert_datetime_to_iso_8601_with_z_suffix(dt: dt.datetime) -> str:
-    return dt.strftime("%Y-%m-%dT%H:%M:%SZ")
+def convert_datetime_to_string(dt: dt.datetime) -> str:
+    return dt.strftime("%Y-%m-%d %H:%M:%S")
```

Comment:
So it turns out this can be a `datetime.fromisostring` even if it has a space instead of a T (and then we don't need this util but use fromisostring instead)

Reply:
I removed this and used `dt.isoformat` with a space as separator